manifests/kustomize/base/installs/multi-user/virtual-service.yaml

```diff
@@ -19,3 +19,27 @@ spec:
         port:
           number: 80
     timeout: 300s
+
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: metadata-grpc
```

Comment:
this was unexpected, let's get this in first to unblock release and I'll verify that afterwards